apps/functions-subscription/api/internal.yaml

```diff
@@ -196,6 +196,37 @@ paths:
               schema:
                 $ref: '#/components/schemas/ProblemJson'
   /trials:
+    get:
+      operationId: listTrials
+      description: List trials
+      summary: List trials
+      parameters:
+        - $ref: '#/components/parameters/queryPageSize'
+        - $ref: '#/components/parameters/queryMaximumId'
+        - $ref: '#/components/parameters/queryMinimumId'
+      responses:
+        '200':
+          description: Success
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/TrialCollection'
```

Comment:
**thought**: how the caller can handle the pagination?

assume we have 100 documents, the `pageSize` query parameter is by default `25`, so the HTTP call will return 25 documents.

do you think would be better to have the next id (the id to use to fetch the next batch of documents) within the response?

Reply:
Added nextId to response in 71323943e0b5b6de4d732d5465c4bafa2973e4b1